snippets/python/src/parsing_inputs.py

```diff
@@ -0,0 +1,56 @@
+import logging
+from breez_sdk_liquid import BindingLiquidSdk, ConnectRequest, InputType, LiquidNetwork, connect, default_config
+
+def parse_input(sdk: BindingLiquidSdk):
+    # ANCHOR: parse-inputs
+    input = "an input to be parsed..."
+
+    try:
+        parsed_input = sdk.parse(input)
+        match parsed_input:
+            case InputType.BITCOIN_ADDRESS:
+                logging.debug(f"Input is Bitcoin address {parsed_input.address.address}")
+            case InputType.BOLT11:
+                amount = "unknown"
+                if parsed_input.invoice.amount_msat:
+                    amount = str(parsed_input.invoice.amount_msat)
+                logging.debug(f"Input is BOLT11 invoice for {amount} msats")
+            case InputType.LN_URL_PAY:
+                logging.debug(f"Input is LNURL-Pay/Lightning address accepting min/max {parsed_input.data.min_sendable}/{parsed_input.data.max_sendable} msats")
+            case InputType.LN_URL_WITHDRAW:
+                logging.debug(f"Input is LNURL-Withdraw for min/max {parsed_input.data.min_withdrawable}/{parsed_input.data.max_withdrawable} msats")
+            # Other input types are available
+    except Exception as error:
+        logging.error(error)
+        raise
+    # ANCHOR_END: parse-inputs
+
+def configure_parsers():
+    # ANCHOR: configure-external-parser
+    mnemonic = "<mnemonic words>"
+
+    # Create the default config, providing your Breez API key
+    config = default_config(network=LiquidNetwork.MAINNET, breez_api_key="<your-Breez-API-key>")
+
+    # Configure external parsers
+    config.external_input_parsers = [
+        {
+            "provider_id": "provider_a",
+            "input_regex": "^provider_a",
+            "parser_url": "https://parser-domain.com/parser?input=<input>"
+        },
+        {
+            "provider_id": "provider_b", 
+            "input_regex": "^provider_b",
+            "parser_url": "https://parser-domain.com/parser?input=<input>"
+        }
```

Comment:
Should we be using the `ExternalInputParser` class here?

Reply:
I somehow missed that. Fix is in [5e26c28](https://github.com/breez/breez-sdk-liquid-docs/pull/56/commits/5e26c281ae6cd126f8cbddbc410608c6dd9c6c03). Thanks